doc/examples/Repository.rst

```diff
@@ -8,4 +8,42 @@ Get repository topics
 
     >>> repo = g.get_repo("PyGithub/PyGithub")
     >>> repo.get_topics()
-    [u'pygithub', u'python', u'github', u'github-api']
\ No newline at end of file
+    [u'pygithub', u'python', u'github', u'github-api']
+
+Get count of stars
+------------------
+
+.. code-block:: python
+
+    >>> repo = g.get_repo("PyGithub/PyGithub")
+    >>> repo.stargrazers_count
+    2086
+
+Get list of open issues
+--------------------------
+
+.. code-block:: python
+
+    >>> repo = g.get_repo("PyGithub/PyGithub")
+    >>> open_issues = repo.get_issues(state='open')
+    >>> for issue in open_issues:
+    ...    print(issue.title)
+    ...
+	How to get public events?
```

Comment:
Ah I see, these are the titles of the issues. Very confusing 😂
Maybe just `print(issue`, it has nicer `repr` format.

Reply:
@sfdye Made changes as requested.

Reply:
You output should be something like this:
```
Issue(title="Add examples for Repository as per #874", number=917)
Issue(title="How to get public events?", number=913)
Issue(title="GithubException: 500 None when using get_issues", number=912)
```

same below

Reply:
@sfdye Done.